src/Poster/GetPoster.js

```diff
@@ -3,7 +3,7 @@ import React, {
 } from 'react';
 import axios from 'axios';
 
-const BASE_URL = 'https://popcorn-studio-17.herokuapp.com';
+const BASEURL = `https://popcorn-studio-17.herokuapp.com`;
```

Comment:
is it a mistake in name of constant?

Reply:
I've changed that because I used BASEURL in my functions, there are any mistakes, just typo style